server/views/components/transporter/index.config.yml

```diff
@@ -0,0 +1,108 @@
+name: Transporter
+handle: transporter
+display:
+  background: "#eeebe2"
+collated: true
+context:
+  title: Latest articles
+  promos:
+  - modifiers: ['transporter-child']
```

Comment:
We should look at this model - as assuming promos are actually just going to be articles (or other types later), which means probably taking a look at the model completely.

Let's bikeshed till we open that CR though.